main.cpp

```diff
@@ -112,9 +225,21 @@ struct Shoe
     int rubberType = 3;
     int agressiveness = 1;
 
-    bool shoeFit( float shoeSize, bool isMale, int painTolerance );
+    bool shoeFit( int painTolerance );
 };
 
+bool Shoe::shoeFit( int painTolerance )
+{
+    bool x = false;
+
+    if ( painTolerance > shoeSize )
+    {
+        x = true;
+    }
+
+    return x;
```

Comment:
you can simplify all of this to : `return painTolerance > shoeSize; `

- [ ] simplify logic.